transactions/updater/delegate.cdc

```diff
@@ -20,21 +20,22 @@ transaction {
         self.delegatee = StagedContractUpdates.getContractDelegateeCapability().borrow()
             ?? panic("Could not borrow Delegatee reference")
 
-        let updaterPrivatePath = PrivatePath(
-                identifier: "StagedContractUpdatesUpdater_".concat(
+        let updaterCapStoragePath = StoragePath(
+                identifier: "StagedContractUpdatesUpdaterCapability_".concat(
```

Comment:
Why did this string change?

Reply:
So here I replaced the deprecated private path with storage path. Since we no longer have private paths by which to manage issued capabilities, I instead store the issued capability in storage for easy traceability and revocation. Ultimately, what's stored at that path is no longer and `Updater` but a capability on an `Updater`, hence the change in path identifier.

Does that capability management scheme make sense or see any problems with that change?

Also, FWIW this contract isn't deployed on TN or MN but updating it here to diminish any tech debt and because the contract may be useful as reference.

Reply:
that all makes sense